freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/FreeIpaV1Endpoint.java

```diff
@@ -80,6 +80,13 @@
             nickname = "listFreeIpaClustersByAccountV1")
     List<ListFreeIpaResponse> list();
 
+    @GET
+    @Path("internal/list")
```

Comment:
why we need /list here ?

Reply:
there is already a GET internal endpoint to describeInternal. adding /list here matches the non-internal endpoints.
/ --> describe
internal --> describeInternal
/list --> list
internal/list --> listInternal

Reply:
in this case it seems different endpoint is needed for internal calls, since original one uses accountid of the user (thus wonn't work in case of internal actor) and we should not break the current list method to add a new parameter and make it mandatory in case of internal call, so it is ok in this case

fyi, if a list method of a given api has a crn or accountid parameter, then different method not needed, just need to use one of annotations mentioned in authorization-common/readme.md regarding internal calls

also fyi, authorization of lists is currently disabled, later list will be filtered baseed on user and we need to prepare filtering logic to skip filtering and return full list based on the account id